{

################# rover.launch ###################

s_port_adaControl               :    /dev/adafruit_serial_rover,
baudrate_adaControl             :    115200,
s_port_adaCblControl            :    /dev/adafruit_serial_cbl,
baudrate_adaCblControl          :    115200,
s_port_adaSensor                :    /dev/ttyACM0,
baudrate_adaSensor              :    115200,

scale_LRight    :    0.3,
scale_FBwards   :    0.5,

pid_kp          :    0.001,
pid_ki          :    0.001,
pid_kd          :    0.005,
pid_intMin      :    -20,
pid_intMax      :    20,


################# cam.launch #######################

rotate          :    True,
rotate_cw       :    True,
flip_vertical   :    False,

########### dual cam ##############
image_provided  :    False,
show_img        :    True,
fcam_input      :    /dev/rovercam_f,
bcam_input      :    /dev/rovercam_b,

}
